fl4health/server/model_merge_server.py

```diff
@@ -0,0 +1,104 @@
+import datetime
+import timeit
+from logging import DEBUG, INFO
+from typing import Optional, Tuple
+
+from flwr.common.logger import log
+from flwr.server.client_manager import ClientManager
+from flwr.server.history import History
+from flwr.server.server import Server
+from flwr.server.strategy import Strategy
+
+from fl4health.reporting.metrics import MetricsReporter
+from fl4health.strategies.model_merge_strategy import ModelMergeStrategy
+
+
+class ModelMergeServer(Server):
+    def __init__(
+        self,
+        *,
```

Comment:
Do you mind reminding me what this * notation does? I think we can limit to just the inputs you have other than the star, but perhaps I'm forgetting something.

Reply:
Placing * in the parameter list makes all subsequent parameters keyword-only. This is done in all of the base classes in flwr (ie Server, Strategy, and Client). I think probably since they have large amounts of input arguments relying on the position is somewhat error prone. Thinking we should probably stick with the flwr convention even though we could technically remove it. Happy to discuss further if you disagree though :)

Reply:
Ah right. Yes, I'm definitely good with that. Thanks for the reminder.